keepassxc-browser/_locales/en/messages.json

```diff
@@ -200,27 +212,31 @@
         "description": "Shown when no username is set in the credentials."
     },
     "credentialsNoLoginsFound": {
-        "message": "Error:\nNo logins found.",
+        "message": "Error No logins found.",
```

Comment:
Did you intend to remove the colon?

Reply:
Yes. But `:` should have stayed. And maybe I'll put these to separate lines later, with just general "Error" message as the first one, and the actual error to the second line.

Reply:
No. And maybe I'll put these to separate lines later, with just general "Error" message as the first one, and the actual error to the second line. I'll fix this soon.

Reply:
>  with just general "Error" message as the first one, and the actual error to the second line

This may be hard to localize, in case some languages put the "Error" somewhere else or so…

Reply:
True.. Maybe it's better to leave just as it is. I'll just add the colon back. (It's frustrating to have these tiny one character errors)